tests/profiles-enabled/test_documents.py

```diff
@@ -112,126 +107,133 @@ def test_sign_to_s3(self):
         response = self.client.post(url, file_data, content_type=content_type)
         self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
 
-        # Authenticated request should succeed
-        self.set_user(self.user_1)
-        response = self.client.post(url, file_data, content_type=content_type)
-        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        with mock.patch.object(rpc_requests.Session, 'post') as mock_method:
+            # mock_method.return_value = mocked_rpc_response({
+            #     "success": True
+            # })
```

Comment:
I do not understand why we need this mock especially since its returned value is commented out

Reply:
This mock is necessary for the check on line 208, where assertTrue(isinstance(data['meta']['presigned_s3'], str)). It is able to work without the mock_method.return_value but it does need the with mock.patch to succeed. We may want to add in further tests later on, or possibly even in this branch, which expects the RPC calls to fails and ensure we get the correct response from them. At the least, I'm gonna change the mock_method in this call to be more explicit, thanks for the catch!